var/spack/repos/builtin/packages/libfabric/package.py

```diff
@@ -133,15 +138,29 @@ def configure_args(self):
 
         return args
 
-    def installcheck(self):
-        fi_info = Executable(self.prefix.bin.fi_info)
-        fi_info()
+    def install(self, spec, prefix):
+        # Call main install method
+        super(Libfabric, self).install(spec, prefix)
 
-        # Build and run more extensive tests
+        # Build and install fabtests, if available
+        if not os.path.isdir('fabtests'):
+            return
         with working_dir('fabtests'):
             configure = Executable('./configure')
             configure('--prefix={0}'.format(self.prefix),
                       '--with-libfabric={0}'.format(self.prefix))
             make()
             make('install')
```

Comment:
We could either do this after install, or alongside the configure/build/install phases. Not sure which would be better.

Reply:
I considered doing it alongside configure/build/install. But fabtests's configure seems to need an _installed_ libfabric. So I think, the order is correct.